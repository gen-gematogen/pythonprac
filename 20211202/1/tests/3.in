class red_bull(metaclass=dump):
	def __init__(self):
		print('Radio cheeeck!')
	def seb(self):
		return "Here comes Sebastion Vettel..."

rb = red_bull()
print(rb.seb())
