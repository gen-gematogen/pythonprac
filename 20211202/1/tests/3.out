__init__: (), {}
Radio cheeeck!
seb: (), {}
Here comes Sebastion Vettel...
